subsys/bluetooth/host/conn_internal.h

```diff
@@ -99,10 +99,23 @@ struct bt_conn_sco {
 struct bt_conn_iso {
 	/* Reference to ACL Connection */
 	struct bt_conn          *acl;
-	/* CIG ID */
-	uint8_t			cig_id;
-	/* CIS ID */
-	uint8_t			cis_id;
+	union {
+		/* CIG ID */
+		uint8_t			cig_id;
+		/* BIG handle */
+		uint8_t			big_handle;
+	};
+
+	union {
+		/* CIS ID */
+		uint8_t			cis_id;
+
+		/* BIS ID */
+		uint8_t			bis_id;
+	};
+
+	/** If true, this is a ISO for a BIS, else it is a ISO for a CIS */
+	bool is_bis;
```

Comment:
We could use the presence of an ACL to indicate if this is a broadcast or a unicast ISO handle.

Reply:
Took me a while to figure out why I didn't just do that. It looks like we can be in a case where acl is NULL, but while it is still a CIS (e.g. when creating CIS'es fails), and this value is currently only being used in the cleanup. So if binding a conn and a iso channel fails, the `acl` pointer cannot be used.